tardis/tests/integration_tests/test_integration.py

```diff
@@ -154,6 +156,34 @@ def test_model_quantities(self, model_quantities):
         tardis_quantity = eval('self.result.' + tardis_quantity_name)
         assert_allclose(tardis_quantity, reference_quantity)
 
+    def plot_model_quantities(self, model_quantities):
+        reference_quantity_name, tardis_quantity_name = model_quantities
+        reference_quantity = self.reference[reference_quantity_name]
+        tardis_quantity = eval('self.result.' + tardis_quantity_name)
+
+        gs_mq = (plt.GridSpec(2, 1, height_ratios=[3, 1]))
+        model_ax_mq = (plt.subplot(gs_mq[0]))
```

Comment:
You don't have to enclose these calls in parenthesis. Same thing applies to other occurrences as well.